apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/MapAssertionSamples.kt

```diff
@@ -0,0 +1,249 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapAssertionSamples {
+
+    @Test
+    fun containsBuilder() {
+        expect(mapOf(1 to "a")) toContain (1 to "a")
+
+        fails {
+            // fails because the map does not contain Pair<1,"b">
```

Comment:
please take a look at MapExpectationSamples.kt, the text has change by now, e.g. here it should be:
```suggestion
        fails { ch.tutteli.atrium.api.infix.en_GB.samples
```
There are also other improvements -- maybe it is simpler to just start over again and copy MapExpectationSamples instead of  MapAssertionSamples?